test/fixtures/gateway-account.fixtures.js

```diff
@@ -15,6 +15,22 @@ function validCredentials (opts = {}) {
   return credentials
 }
 
+function validGatewayAccountCredential (credentialOpts = {}, gatewayAccountOpts = {}) {
+  const gatewayAccountCredential = {
+    gateway_account_credential_id: credentialOpts.id || 1,
+    payment_provider: gatewayAccountOpts.payment_provider || 'sandbox',
+    state: credentialOpts.state || 'ACTIVE',
+    gateway_account_id: gatewayAccountOpts.gateway_account_id || 31,
+    // '2021-05-10T01:00:00.000Z'
```

Comment:
what's this comment for?

Reply:
This was another thing that was originally default but isn't guaranteed from the provider, the comment shows what format it expects but I forgot it was in there to be honest, probably isn't needed